src/compiler/types.ts

```diff
@@ -4120,6 +4120,7 @@ namespace ts {
         isFilePath?: boolean;                                   // True if option value is a path or fileName
         shortName?: string;                                     // A short mnemonic for convenience - for instance, 'h' can be used in place of 'help'
         description?: DiagnosticMessage;                        // The message describing what the command line switch does
+        descriptionOnTSConfig?: DiagnosticMessage;              // The message describing what the option does in tsconfig.json file

```

Comment:
Is it okay to add new property here?
I think it has a benefit: we can also fix more command-line specific messages in `tsconfig.json`, like:

- Specify the root directory of input files. Use to control the output directory structure with `--outDir`.
- Emit the source alongside the sourcemaps within a single file; requires `--inlineSourceMap` or `--sourceMap` to be set.

We can use non-double-dash version for `tsconfig.json`.

Reply:
I think this is an overkill to be frank. We can make the message look the same in both.